### PR TITLE
EDTT: inclusive naming 

### PR DESCRIPTION
**Pull request by kruithofa**
- Remove trailing whitespace in python scripts
- Inclusive naming changes:
    'master' is replaced with 'central'
    'slave' is replaced with 'peripheral'
    'wl' is replaced with 'fal' (filtered accept list)
    'whitelist' is replaced with 'acceptlist'
- Rename .csv file to be compliant with inclusive naming convention
    
Signed-off-by: kruithofa <Andries.Kruithof@nordicsemi.no>

### CODE REVIEW

File: src/tests/ll_verification.py [hunk 1]
```diff
@@ -5939,58 +5939,58 @@ def ll_sec_scn_bv_01_c(transport, upperTester, lowerTester, trace):
...
-    "LL/CON/SLA/BV-29-C": [ ll_con_sla_bv_29_c, "Slave responds to Connection Parameters - Master no Preferred Periodicity" ],
-    "LL/CON/SLA/BV-33-C": [ ll_con_sla_bv_33_c, "Slave responds to Connection Parameters request - event masked" ],
-    "LL/CON/SLA/BV-34-C": [ ll_con_sla_bv_34_c, "Slave responds to Connection Parameters request - Host rejects" ],
-    "LL/CON/SLA/BV-40-C": [ ll_con_sla_bv_40_c, "Slave requests PHY Update procedure" ],
-    "LL/CON/SLA/BV-42-C": [ ll_con_sla_bv_42_c, "Slave responds to PHY Update procedure" ],
-    "LL/CON/SLA/BV-77-C": [ ll_con_sla_bv_77_c, "Slave Packet Data Length Update - Pesponding to Packet Data Length Update Procedure; LE 1M PHY" ],
-    "LL/CON/SLA/BV-78-C": [ ll_con_sla_bv_78_c, "Slave requests Packet Data Length Update procedure; LE 1M PHY" ],
-    "LL/CON/SLA/BV-80-C": [ ll_con_sla_bv_80_c, "Slave Packet Data Length Update - Pesponding to Packet Data Length Update Procedure; LE 2M PHY" ],
-    "LL/CON/SLA/BV-81-C": [ ll_con_sla_bv_81_c, "Slave requests Packet Data Length Update procedure; LE 2M PHY" ],
+    "LL/CON/CEN/BI-06-C": [ ll_con_cen_bi_06_c, "Central responds to Connection Parameter Request - illegal parameters" ],
```
Comment (aescolar) [inline src/tests/ll_verification.py:5944]: Maybe a silly question: the testcase names and descriptions (here and above in each test) had been extracted from the test spec automatically.
I guess you have checked that you are doing the same renaming as what happened in the spec?

Comment (kruithofa) [reply src/tests/ll_verification.py:5944]: Very good question.
I admit I didn't do a 100% verification of testcase names, but on the 10-20 cases I checked the renaming is the same as in the spec

File: src/tests/hci_verification.py [hunk 2]
```diff
@@ -249,18 +249,18 @@ def hci_cin_bv_06_c(transport, idx, trace):
     trace.trace(6, "LE Clear White List Command returns status: 0x%02X" % status);
```
Comment (wopu-ot) [inline src/tests/hci_verification.py:249]: Here and other places, "white list" should be replaced with "filter accept list"

**Review by wopu-ot — CHANGES_REQUESTED** [review]

**Review by wopu-ot — APPROVED** [review]

**Comment by thoh-ot** [comment]
There seems to be a conflict that needs resolving

**Review by thoh-ot — APPROVED** [review]

**Review by wopu-ot — APPROVED** [review]

**Pull request merged**